app/cli/Main.hs

```diff
@@ -130,7 +136,11 @@ exec args
         print (wid :: WalletId)
 
     | args `isPresent` command "wallet" && args `isPresent` command "list" = do
-        return ()
+        walletPort <- args `parseArg` longOption "port"
+        res <- runClientM listWallets (mkClientEnv manager (BaseUrl Http "localhost" walletPort ""))
```

Comment:
@KtorZ 

what do you think about this approach vs the one we have in httpBridge https://github.com/input-output-hk/cardano-wallet/blob/master/src/Cardano/Wallet/Network/HttpBridge.hs#L167 where we are wrapping client calls to a record and having a smart constructor in `main` which will construct this record to talk to the backend (we could call it `WalletClient` or similar).

With this one we are passing `Manager` to the `exec` where with the `HttpBridge` like we would defined `WalletClient`, defined the api calls in there and then create an instance of this in `main` and pass it to `exec`.